bundles/org.eclipse.cdt.lsp.editor.ui.test/META-INF/MANIFEST.MF

```diff
@@ -7,6 +7,6 @@ Bundle-Vendor: Eclipse.org
 Fragment-Host: org.eclipse.cdt.lsp.editor.ui;bundle-version="0.0.0"
 Require-Bundle: org.junit,
  org.eclipse.cdt.lsp,
- junit-jupiter-api
+ org.junit.jupiter.api
```

Comment:
In case you didn't know, the reason that the bundle name changed is that in 2022-06 the Jupiter API came from Orbit, now it comes directly from Maven.

To easily support both versions we can do:

`
  Import-Package: org.junit.jupiter.api;version="5.8.0"
`

with an appropriate minimum version, probably 5.8.0 as that was in 2022-06.